### PR TITLE
Rename offline-page to offline-crossword

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

prepares space for alternative offline offering (for now, a simple cloudwatch ping for the rest of the site)

_update_
some context for this: getting metrics from the current offline page (by stroring a view in localstorage and pinging cloudwatch if we detect it again) limits us to users of the dev blog, who go offline, then come back to another secure page on the guardian subsequently. 

this clears out the old metrics stuff (which didn't work) and makes space for a metrics-only offline page that can be served regardless of the section, but which can leave the current crossword in place for dev blog
## What is the value of this and can you measure success?

makes it easier to add more than one offline treatment
## Request for comment

@OliverJAsh @rich-nguyen

### CODE REVIEW

File: applications/app/templates/serviceWorker.scala.js [hunk 1]
```diff
@@ -23,11 +20,11 @@ var fetchAll = function (inputs) {
     }));
 };
 
-var cachePageAndAssetResponses = function (jsonResponse, assetResponses) {
```
Comment (OliverJAsh) [inline applications/app/templates/serviceWorker.scala.js]: Maybe you could name space all of the offline crossword code by wrapping them in an IIFE? It would be good to group the notifications stuff in the same way!

``` js
// Offline crossword
(() => {
  var updateCache = () => {};
})();
```

**Comment by OliverJAsh** [comment]
Looks fine to me! 👍  I'm happy to help testing this locally if needed

File: applications/app/templates/serviceWorker.scala.js (hunk 1 above)
Comment (sndrs) [inline applications/app/templates/serviceWorker.scala.js]: yeah good thinking, I will do

Comment (sndrs) [inline applications/app/templates/serviceWorker.scala.js]: i'm going to do this in a separate PR, with a bit of further rejigging (caching regex etc)
